db/schema.rb

```diff
@@ -54,6 +96,143 @@
     t.string "zip", limit: 5
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.string "name"
+    t.string "description"
+    t.string "region"
+    t.string "neighborhood"
+    t.string "county"
+    t.string "facebook_url"
+    t.string "website_url"
+    t.string "phone"
+    t.string "location_type"
+    t.bigint "parent_id"
+    t.index ["parent_id"], name: "index_locations_on_parent_id"
```

Comment:
I was confused also, but this is what I understood was being recommended in a prior discussion. 

I don't understand the value of creating Locations as a separate model rather than having address-related files on Person, Organization(and/or)ExternalResource.

SystemLocation makes sense to me as a standalone bc it's what will be selected from when the public fill out Ask/Offer forms and/or ExternalResource submissions, and what show up as filters on the Share page.